diagnostic.md

```diff
@@ -35,5 +42,7 @@ Record your responses inside the fenced code blocks below each question.
     pass it data?
 
     ```html
-    <!-- your response here -->
+    {{#each contact.numbers as |number|}}
+      {{contacts/my-contact/my-phone number=number}}
+    {{/each}}
```

Comment:
you don't need the `contacts/my-contact/`.  So it will look like this:
```
{{#each contact.phones as |phone|}}
{{my-phone phone=phone}}
{{/each}}
```